packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -264,4 +267,105 @@ exports.sourceNodes = async (
   for (const node of nodes) {
     createNode(node)
   }
+
+  if (process.env.NODE_ENV === `development` && preview) {
```

Comment:
Curious if there's any way to unify logic between here & the regular fetches?

Reply:
could you elaborate?